pkg/appender/store.go

```diff
@@ -326,8 +339,10 @@ func (cs *chunkStore) writeChunks(mc *MetricsCache, metric *MetricState) (hasPen
 			// Add a value to the aggregates list
 			cs.aggrList.Aggregate(sampleTime, cs.pending[pendingSampleIndex].v)
 
-			// Add a value to the compressed raw-values chunk
-			activeChunk.appendAttr(sampleTime, cs.pending[pendingSampleIndex].v)
+			if activeChunk != nil {
```

Comment:
even though it's basically the same, i would consider changing the condition to `!cs.isAggr()` because this is the real case you want to append stuff to the chunk. so it might be more clear.